AsyncDisplayKit/Private/ASDisplayNodeInternal.h

```diff
@@ -22,6 +22,13 @@ BOOL ASDisplayNodeSubclassOverridesSelector(Class subclass, SEL selector);
 CGFloat ASDisplayNodeScreenScale();
 void ASDisplayNodePerformBlockOnMainThread(void (^block)());
 
+typedef NS_OPTIONS(NSUInteger, ASMethodOverrides) {
```

Comment:
nits:  let's name it `ASDisplayNodeMethodOverrides` and add a `...None` instead of using 0 directly